VicenteDouriet/ChallengesDay26/Hackerrank/SolutionPrintf/SolutionPrintf.java

```diff
@@ -0,0 +1,17 @@
+import java.util.Scanner;
+
+public class SolutionPrintf {
+
+   public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        String equalsInALine = "================================"
+        System.out.println(equalsInALine);
+        for (int i = 0; i < 3; i++) {
```

Comment:
Variables names must be meaningful 

Avoid using magic numbers and magic strings